### DISCUSSION TITLE
Arbitrary value for box-shadow not working on production build

### DISCUSSION THREAD

**Discussion by WillyReyno** [tailwindlabs/tailwindcss#8440] [Help]
**What version of Tailwind CSS are you using?**

v3.0.24

**What build tool (or framework if it abstracts the build tool) are you using?**

Laravel Mix 6.0.43, Webpack 4.9.2

**What version of Node.js are you using?**

v14.18.3

**What browser are you using?**

Google Chrome 101.0.4951.64

**What operating system are you using?**

macOS Monterey 12.3.1 

**Reproduction URL**

https://play.tailwindcss.com/0RnRxLIvem, but not sure it will help since there's no production build.

## Description

When using arbitrary value for box-shadow and building assets with `yarn production`, the class is not applied while it works with `yarn dev`

Here's the code I had when I noticed this behaviour : 
```
<div class="relative h-full bg-white rounded-lg shadow-[0_0_10px_rgba(205,205,205,1)] overflow-hidden flex flex-row xl:flex-col">
...
</div>
```

EDIT : just noticed the same with `w-[2.6rem]`, maybe something is wrong in my configuration?

**Comment by adamwathan** [comment]
Hey! Can you provide a reproduction that actually reproduces your issue, like a GitHub repo that includes your production build steps? This definitely should work (works for me in our sites anyways) so I’m sure we can figure this out for your project! 🤞

**Reply by WillyReyno** [reply]
In the end I found that I had left a `.purgeCss()` from `laravel-mix-purgecss` in my `webpack.mix.js` which was causing the issue. 

Thanks for your help!
